microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md

```diff
@@ -89,6 +89,28 @@ You can define mail flow rules for triggering message encryption with the new OM
 
 8. Choose **Save**.
 
+## Create mail flow rules to remove encryption for outgoing email messages with the new OME capabilities
```

Comment:
Create a mail flow rule to remove encryption for outgoing email message by using the new OME capabilities